parsers/ZA.py

```diff
@@ -0,0 +1,962 @@
+from __future__ import annotations
+
+from arrow import now, get
+from collections import OrderedDict
+from datetime import datetime
+from logging import Logger, getLogger
+from pprint import PrettyPrinter
+from requests import Response, post, Session
+from requests import session as req_session
```

Comment:
```suggestion
```

Reply:
Done in commit 355a3467b0951490eb00d5d08ff2c31f6be1dd74